assets/differentialabundance_report.Rmd

```diff
@@ -937,6 +1014,12 @@ if (any(unlist(params[paste0(possible_gene_set_methods, '_run')]))){
 }
 ```
 
+```{r, echo=FALSE, results='asis', eval=params$gprofiler2_run}
```

Comment:
This versions table should happen automatically as part of the above - you shouldn't need a new block